webconsole/src/main/java/org/secondbase/webconsole/WebConsoleConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package org.secondbase.webconsole;
+
+import org.secondbase.flags.Flag;
+
+/**
+ * Configuration parameters for the WebConsole.
+ */
+public class WebConsoleConfiguration {
+    @Flag(name = "enable-webconsole", description = "Start the web console")
```

Comment:
Empty line before this and in between the individual flags to be more consistent with other classes?